lib/broccoli/angular-broccoli-compass.js

```diff
@@ -0,0 +1,66 @@
+/* jshint node: true, esversion: 6 */
+'use strict';
+
+let sass;
+let compass;
+try {
+  try {
+    sass = require('sass');
+    compass = require('compass');
+  } catch (e) {
+    sass    = require(`${process.env.PROJECT_ROOT}/node_modules/node-sass`);
```

Comment:
You cannot require peer dependencies without having a peerDependency key in package.json.  We could put it there as optional. We need to experiment on this.